package.json

```
@@ -30,7 +30,7 @@
     "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
     "test:ampValidate": "npm run killApp && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
     "test:ampValidate:ci": "npm run build -- --hide-modules --colors && run-p --race start amp:validate",
-    "test:prod:ci": "npm install --no-save cypress@3.1.5 bbc-a11y@2.2.6 npm-run-all@4.1.5 && run-p --race start cypress && run-p --race start bbcA11y",
+    "test:prod:ci": "npm install cypress@3.1.5 bbc-a11y@2.2.6 npm-run-all@4.1.5 && run-p --race start cypress && run-p --race start bbcA11y",
```

`--no-save` is needed to ensure it doesn't add these packages to `dependencies`
```suggestion
    "test:prod:ci": "npm install --no-save cypress@3.1.5 bbc-a11y@2.2.6 npm-run-all@4.1.5 && run-p --race start cypress && run-p --race start bbcA11y",
```